src/core/components/parameter-row.jsx

```diff
@@ -181,6 +183,13 @@ export default class ParameterRow extends Component {
     return `${param.get("name")}-${param.get("in")}`
   }
 
+  mergeJsonSchema(schema) {
```

Comment:
What's happening here is a schema composition. The merging that is happening under the hood is just an implementation detail.

```suggestion
  composeJsonSchema(schema) {
```